wsdl/ver10/schema/onvif.xsd

```diff
@@ -3828,6 +3828,22 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 		</xs:restriction>
 	</xs:simpleType>
 	<!--===============================-->
+	<xs:complexType name="UserRole">
+		<xs:sequence>
+			<xs:element name="Name" type="xs:string">
+				<xs:annotation>
+					<xs:documentation>Name of the editable user level.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="Functions" type="tt:StringList">
+				<xs:annotation>
```

Comment:
For the predefined userroles like onvif:Administrator etc., the functions parameter can be empty ? or do we have fill this for pre-defined userroles also ? can we clarify this somewhere.